source/glest_game/menu/menu_state_load_game.cpp

```diff
@@ -317,16 +318,18 @@ void MenuStateLoadGame::mouseClick(int x, int y, MouseButton mouseButton){
 
 #endif
 
-						XmlTree	xmlTree(engine_type);
-
 						if(SystemFlags::VERBOSE_MODE_ENABLED) printf("Before load of XML\n");
+                        versionWarningLabel.setText("");
+                        infoTextLabel.setText("Loading...");
```

Comment:
Spaces vs tabs, this codebase uses mostly tabs, let's try to keep it consistent :)